pages/dashboard/v2/[id].js

```diff
@@ -13,6 +14,30 @@ export async function getServerSideProps(context) {
     return {};
   }
 
+  const userEmail = await prisma.User.findMany({
```

Comment:
I forgot, do we need to findMany here because email is not primary key? 

We should figure out later how to make email a primary key, we can discuss sun

Reply:
OK!